dvc/utils/fs.py

```diff
@@ -35,11 +34,11 @@ def get_inode(path):
     return inode
 
 
-def get_mtime_and_size(path, dvcignore):
+def get_mtime_and_size(path, tree):
```

Comment:
Need to assert that tree is a working tree/clean working tree. Passing git tree here will break the abstraction - we walk git, but `os.stat()` working tree.